api/spec/v1.yaml

```diff
@@ -2113,23 +2113,28 @@ components:
         verification:
           $ref: '#/components/schemas/RuntimeEvmContractVerification'
           description: |
-            Additional information obtained from contract verification. Only available for smart 
+            Additional information obtained from contract verification. Only available for smart
             contracts that have been verified successfully by Sourcify.
-    
+
     RuntimeEvmContractVerification:
       type: object
-      # properties:
-      #   abi:
-      #     type: object
-      #     description: |
-      #       The smart contract's [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI), in JSON format as defined by Solidity. Available if 
-      #       and only if the contract has been "verified" through [Sourcify](https://sourcify.dev), i.e. Sourcify has checked 
-      #       and confirmed that the source code and ABI correspond to the contract's creation bytecode.
-      #   compiler_version:
-      #     type: string
-      #     description: |
-      #       The version of the compiler used to verify the smart contract.
-      #     example: 0.8.18+commit.87f61d96
+      properties:
+        abi:
+          type: array
+          items:
+            type: object
+          description: |
+            The smart contract's [ABI](https://docs.soliditylang.org/en/develop/abi-spec.html), in JSON format as defined by Solidity.
+        metadata_json:
+          type: object
+          description: |
+            The smart contract's [metadata.json](https://docs.soliditylang.org/en/latest/metadata.html) file in JSON format as defined by Solidity.
+        source_files:
+          type: array
+          items:
+            type: object
+          description: |
+            Array of all contract source files, in JSON format as returned by [Sourcify](https://docs.sourcify.dev/docs/api/server/get-source-files-full/).
```

Comment:
Super nice to have all the links!

Nit: Looks like this page was just recently removed (Google points to the same broken URL :)) in favor of [the default openapi rendering](https://sourcify.dev/server/api-docs/#/Repository/get_files_any__chain___address_)